pymatgen/analysis/phase_diagram.py

```diff
@@ -208,23 +199,62 @@ class TransformedPDEntry(PDEntry):
     compositions.
     """
 
-    def __init__(self, comp, original_entry):
+    # Tolerance for determining if amount of a composition is positive.
+    amount_tol = 1e-5
+
+    def __init__(self, entry, sp_mapping, name=None):
```

Comment:
`__init__` for `TransformedEntry` has also changed substantially, could you clarify what these changes are for?

Reply:
Previously the `TransformedPDEntry` didn't give an indication as to what the terminal species were. As a result I brought a section of code from the phase diagram inside the entry so that it could be used in more of a standalone manner if ever needed.